compat/src/Children.js

```diff
@@ -1,7 +1,7 @@
 import { toChildArray } from 'preact';
 
 const mapFn = (children, fn) => {
-	if (!children) return null;
+	if (children === undefined || children === null) return null;
```

Comment:
Found a shorter equivalent, thanks for notifying us about this bug!

```suggestion
	if (children == null) return null;
```